src/RealtimeEffectPanel.cpp

```diff
@@ -822,6 +902,7 @@ class RealtimeEffectListWindow : public wxScrolledWindow
    void ResetTrack()
    {
       mEffectListItemMovedSubscription.Reset();
+      mPluginsChangedSubscription.Reset();
```

Comment:
The constructor makes subscriptions.

`ResetTrack()` destroys both subscriptions.

`SetTrack()` restores the old subscription to non-null, but not the new one.

Is that right or wrong?

Reply:
`mPluginsChangedSubscription.Reset()` shouldn't be here, thanks!